front-end/src/components/ui/FormInputField.tsx

```diff
@@ -61,7 +61,7 @@ export const FormInputField: React.FC<{ id: string }> = ({ id }) => {
                   setText ? setText(e.target.value) : ""
                 }}
                 type="text"
-                className={`${inputBorderColor} focus:outline-none`}
+                className={`${inputBorderColor} focus:outline-none ${align.startsWith("OS") ? align.split("_")[1] : "w-full"}`}
```

Comment:
Anstatt mit tailwind `safelist` zu arbeiten würde ich hier ausnahmsweise einen inline-style benutzen.
Also die FormInputField kriegt als zusätzliche prop den Parameter `width?: int`, der die Breite in gewünschten Zeichen (in der Einheit `em` oder `ch`) darstellt. Der Code wäre dann sowas wie

```
<Input
...
classname={`${inputBorderColor} focus:outline-none`}
style={ width !== undefined ? {width: `${width}em`} : undefined }
...>
```

In der Definition von <Input> (`input.tsx`) wird `w-full` bereits gesetzt, und der inline-style überschreibt das, wenn er gesetzt ist.